skbio/stats/power.py

```diff
@@ -49,7 +49,6 @@
     subsample_power
     subsample_paired_power
     confidence_bound
-    bootstrap_power_curve
```

Comment:
Can you add this back? It'll get removed when the deprecated function is removed in 0.3.1.